### PR TITLE
EZP-27445: Use action autowiring rather than constructor in controller

### PR DESCRIPTION
**Pull request by maxbaldanza**

### CODE REVIEW

File: src/bundle/Resources/config/views.yml [hunk 1]
```diff
@@ -13,5 +13,5 @@ system:
             versions_tab:
                 default:
                     template: "EzSystemsHybridPlatformUiBundle:content/tabs:versions.html.twig"
-                    controller: "EzSystemsHybridPlatformUiBundle:ContentView:versionsTab"
+                    controller: "EzSystems\\HybridPlatformUiBundle\\Controller\\ContentViewController:versionsTabAction"
```
Comment (bdunogier) [inline src/bundle/Resources/config/views.yml]: I'd rather use single quotes to avoid escaping the backslash.

**Review by bdunogier — CHANGES_REQUESTED** [review]
I'd change the definition of the VersionFilter service while at it.

Change:
```yaml
    ezsystems.platform_ui.hybrid.filter.version_filter:
        class: EzSystems\HybridPlatformUi\Filter\VersionFilter
```
to
```yaml
    EzSystems\HybridPlatformUi\Filter\VersionFilter: ~
```

File: src/bundle/Resources/config/views.yml (hunk 1 above)
Comment (maxbaldanza) [reply src/bundle/Resources/config/views.yml]: Sure, was just trying to keep it consistent but happy to update 👍

Comment (bdunogier) [reply src/bundle/Resources/config/views.yml]: Valid argument, but I still think that we should go for single quotes. We could make it consistent by going single quotes everywhere. But then I'd prefer a distinct commit, if we can keep the PRs simple.

**Review by bdunogier — APPROVED** [review]
Great.

**Review by mcurran-inviqa — APPROVED** [review]

**Pull request merged**